mozjs-sys/build.rs

```diff
@@ -287,6 +305,9 @@ fn build_spidermonkey(build_dir: &Path) {
         }
         cmd.env("PKG_CONFIG_PATH", pkg_config_path);
     }
+    if let Ok(include) = std::env::var("DEP_Z_INCLUDE") {
+        cppflags.push(format!("-I{include}").replace("\\", "/"));
+    }
```

Comment:
What's this path used for? I wonder if the Windows error is caused by the absolute path when building Spidermonkey.